docs/src/quickstart/index.md

```diff
@@ -0,0 +1,307 @@
+
+Click the `Settings`(gear) icon along the top of the Docker app.
+
+![Settings icon in Docker`](/qs/settings-icon.png)
+
+Verify that `WSL 2.0` is checked, as well as `Use Docker Compose V2`.
+
+![WSL 2 settings in Docker`](/qs/wsl-2.png)
+![Docker Compose in Docker`](/qs/docker-compose.png)
+
+Click `Resources` in the left-hand menu, and check that `Enable integration with additional distros` is checked, and that `Ubuntu` is selected.
+
+![WSL integration in Docker`](/qs/wsl-integrate.png)
+
+Double-click the Ubuntu icon in the Windows Start-up menu to open the Ubuntu terminal.
+You will need to provide a `username` and `password` for Ubuntu.
+
+In the terminal, run the following to install `make`:
+
+```cmd
+$ sudo apt install make
+```
+
+Next, run the following to allow `apt` to use repositories containing https:
+
+```cmd
+$ sudo apt-get install ca-certificates curl gnupg lsb-release
+```
+
+Create and navigate to the reach directory with the following command:
+
+``` cmd
+$ mkdir -p ~/reach && cd ~/reach
+```
+
+Download Reach with the following command:
+
+``` cmd
+$ curl https://docs.reach.sh/reach -o reach ; chmod +x reach
+```
+
+Reach is successfully downloaded with the following command returns a version number:
+
+```
+$ ./reach version
+```
+
+You are now ready to start programming in Reach.
+Check out our [Rock Paper Scissors!](##tut) tutorial to get started.
+
+# (#ref-qs-linux) Linux
+
+Reach requires [make](https://en.wikipedia.org/wiki/Make_(software)), [Docker Engine](https://docs.docker.com/get-docker/), and [Docker Compose](https://docs.docker.com/compose/install/).
+
+## (#ref-qs-linux-prereqs) Prerequisites
+
++A compatible version of Linux.
+Check the [Docker Engine](https://docs.docker.com/engine/install/) page for supported distros. 
+
+## (#ref-qs-linux-install) Installation
+
+Follow the [Docker Engine](https://docs.docker.com/engine/install/) instructions for installing on your version of Linux.
+
+In the terminal, run the following to install `make`:
+
+```cmd
+$ sudo apt install make
+```
+
+Next, run the following to allow `apt` to use repositories containing https:
+
+```cmd
+$ sudo apt-get install ca-certificates curl gnupg lsb-release
+```
+
+After that, run:
+
+```cmd
+$ sudo curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /usr/share/keyrings/docker-archive-keyring.gpg
+```
+
+And then, run:
+
+```cmd
+$ sudo echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/docker-archive-keyring.gpg] https://download.docker.com/linux/ubuntu \ $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+```
+
+Create and navigate to the reach directory with the following command:
+
+``` cmd
+$ mkdir -p ~/reach && cd ~/reach
+```
+
+Download Reach with the following command:
+
+``` cmd
+$ curl https://docs.reach.sh/reach -o reach ; chmod +x reach
+```
+
+Reach is successfully downloaded with the following command returns a version number:
```

Comment:
```suggestion
Reach is successfully downloaded when the following command returns a version number:
```

Reply:
I updated all three references of this to "if"